diff --git a/node_modules/react-native-reanimated/android/src/main/cpp/NativeProxy.cpp b/node_modules/react-native-reanimated/android/src/main/cpp/NativeProxy.cpp
index 96c8022..6d19cce 100644
--- a/node_modules/react-native-reanimated/android/src/main/cpp/NativeProxy.cpp
+++ b/node_modules/react-native-reanimated/android/src/main/cpp/NativeProxy.cpp
@@ -473,6 +473,7 @@ void NativeProxy::setGlobalProperties(
 
   jsRuntime.global().setProperty(
       jsRuntime, "_WORKLET_RUNTIME", workletRuntimeValue);
+  jsRuntime.global().setProperty(jsRuntime, "_WORKLET", false);
 
 #ifdef RCT_NEW_ARCH_ENABLED
   jsRuntime.global().setProperty(jsRuntime, "_IS_FABRIC", true);
diff --git a/node_modules/react-native-reanimated/ios/REAModule.mm b/node_modules/react-native-reanimated/ios/REAModule.mm
index 3cb75a0..7c6dfa3 100644
--- a/node_modules/react-native-reanimated/ios/REAModule.mm
+++ b/node_modules/react-native-reanimated/ios/REAModule.mm
@@ -214,6 +214,7 @@ RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(installTurboModule)
     workletRuntimeData[0] = reinterpret_cast<uintptr_t>(reanimatedModule->runtime.get());
 
     runtime.global().setProperty(runtime, "_WORKLET_RUNTIME", workletRuntimeValue);
+    runtime.global().setProperty(runtime, "_WORKLET", false);
 
     runtime.global().setProperty(runtime, "_IS_FABRIC", true);
 
diff --git a/node_modules/react-native-reanimated/ios/native/REAInitializer.mm b/node_modules/react-native-reanimated/ios/native/REAInitializer.mm
index 60ee16d..927013f 100644
--- a/node_modules/react-native-reanimated/ios/native/REAInitializer.mm
+++ b/node_modules/react-native-reanimated/ios/native/REAInitializer.mm
@@ -56,6 +56,7 @@ JSIExecutor::RuntimeInstaller REAJSIExecutorRuntimeInstaller(
     workletRuntimeData[0] = reinterpret_cast<uintptr_t>(reanimatedModule->runtime.get());
 
     runtime.global().setProperty(runtime, "_WORKLET_RUNTIME", workletRuntimeValue);
+    runtime.global().setProperty(runtime, "_WORKLET", false);
 
     runtime.global().setProperty(runtime, "_IS_FABRIC", false);
 
